### PR TITLE
add magicsuggest v2.1.4

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/nicolasbize/magicsuggest
**Watch 76
  Star 1,121
 Fork 202**
@Amomo could u help me check this PR for #6661 ? thank you!
because i thought this repo is a dist, so i use `*.+(js|css|map)` in auto-update config. 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/magicsuggest/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    "dropdown",
+    "multiselect",
+    "tag",
+    "tagging",
+    "bootstrap",
+    "combobox"
+  ],
+  "author": {
+    "name": "Nicolas Bize",
+    "url": "https://github.com/nicolasbize/magicsuggest"
```
Comment (PeterDaveHello) [inline ajax/libs/magicsuggest/package.json]: What's the reason why to point to the repo?

**Comment by maruilian11** [comment]
@Amomo i have corrected `author` url, could you help me check this PR? thanks.

File: ajax/libs/magicsuggest/package.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    "name": "Nicolas Bize",
+    "url": "https://github.com/nicolasbize"
+  },
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nicolasbize/magicsuggest.git",
+    "basePath": "",
+    "files": [
+      "*.+(js|css|map)"
```
Comment (PeterDaveHello) [inline ajax/libs/magicsuggest/package.json]: Wildcard in non-dist repo's root?

**Comment by maruilian11** [comment]
@PeterDaveHello i have updated the auto-update config, but how do i tell if the repo is a dist repo or not?

**Pull request merged**